task/config.go

```diff
@@ -54,8 +57,5 @@ func (c Config) Validate() error {
 	if len(c.TaskRunOrgID) > 0 && len(c.TaskRunOrg) > 0 {
 		return fmt.Errorf("only one of task-run-org and task-run-orgid should be set")
 	}
-	if len(c.TaskRunBucket) == 0 {
-		return fmt.Errorf("task-run-bucket is required")
-	}
```

Comment:
Why is this check no longer necessary? What if the config file had `task-run-bucket =""`?

Reply:
In that case we want to use the default - otherwise people who generated a default config from 1.6.1 would not get auto-created buckets.

Reply:
That's what I thought, thanks for confirming